src/outputs/hls_output.ml

```diff
@@ -36,10 +53,21 @@ let hls_proto kind =
      Some (Lang.float 10.),
      Some "Segment duration (in seconds).";
 
+     "segment_name",
+     segment_name_t,
+     Some default_name,
+     Some "Segment name. \
+           Default: `fun (~duration:_,~position,~extname,stream_name) -> \"#{stream_name}_#{position}.#{extname}\"`";
+
      "segments",
      Lang.int_t,
+     Some (Lang.int 15),
```

Comment:
I would do everything in terms of duration rather than segments.

Reply:
What's your rationale for that?

Reply:
While using duration could make sense, I think it's less user friendly because it need to be a multiple of the segment duration.

Reply:
My idea was that we want to specify how much time back we want to keep rather than a number of segments (for instance if you double the duration of segments you are likely to want to halve the number of segments to keep). But this is rather minor.